Elmah.Io.ElasticSearch/ElasticClientSingleton.cs

```diff
@@ -18,9 +18,13 @@ public class ElasticClientSingleton : IDisposable
         private static ElasticClientSingleton _instance;
         public IElasticClient Client;
 
-        private ElasticClientSingleton()
+        private ElasticClientSingleton(IDictionary config = null)

```

Comment:
@TrymBeast cool, then I think if you remove that optional param we'll be good here.